<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domain11</class>
 <widget class="QWidget" name="domain11">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>preamble :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditPreamble">
          <property name="inputMask">
           <string>bbbbbbbbbbbbbbbbbbbbbbbb</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>id :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditId">
          <property name="inputMask">
           <string>bbbbbbbbbbbbbbbb</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>type :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditType">
          <property name="inputMask">
           <string>bbbb</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>sample(MHz) : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditSample">
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>sync rate(MHz) : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditSyncRate">
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnConfirm11">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
